pkg/datapath/loader/base.go

```diff
@@ -247,7 +247,7 @@ func (l *Loader) Reinitialize(ctx context.Context, o datapath.BaseProgramOwner,
 	args = make([]string, initArgMax)
 
 	sysSettings := []sysctl.Setting{
-		{Name: "net.core.bpf_jit_enable", Val: "1", IgnoreErr: true},
+		{Name: "net.core.bpf_jit_enable", Val: "1", IgnoreErr: true, Warn: "Could not enable BPF JIT compilation. This can be ignored when running inside non-host network namespace (KinD, Minikube)"},
```

Comment:
"Could not enable BPF JIT compilation" sounds a bit odd to me. In kind or minikube case, we even can't know if it is already enabled or not right? In that case, just telling the fact like "Unable to access to /proc/sys/net/core/bpf_jit_enable" should be sufficient I think.

Reply:
While we're at it, kind upstream just uses `kind` rather than `KinD`, even if that's where then name came from. Should keep consistent with them.

Reply:
Thanks, applied the suggestions!